src/main/java/org/fundacion/jala/converter/models/parameter/AudioParameter.java

```diff
@@ -125,67 +134,78 @@ public String formatAudioChannel() {
                 return " -ac " + SIX_POINT_ONE_CHANNEL + " ";
             case "7.1":
                 return " -ac " + SEVEN_POINT_ONE_CHANNEL + " ";
-                default:
-                    return "";
+            default:
+                return "";
         }
     }
 
     /**
-     * Obtains the format of audio converter
-     * @return a string of format of audio converter
+     * Obtains the format of audio converter.
+     *
+     * @return a string of format of audio converter.
      */
     public String getFormat() {
         return format;
     }
 
     /**
-     * Sets new format of the audio converter
-     * @param newFormat
+     * Sets new format of the audio converter.
+     *
+     * @param newFormat is a new format to audio.
      */
     public void setFormat(final String newFormat) {
         this.format = newFormat;
     }
+
     /**
-     * Obtains bitrate of audio converter
-     * @return a string of bitrate of audio converter
+     * Obtains bitrate of audio converter.
+     *
+     * @return a string of bitrate of audio converter.
      */
     public String getBitrate() {
         return bitrate;
     }
 
     /**
-     * Sets new bitrate of the audio converter
-     * @param newBitrate
+     * Sets new bitrate of the audio converter.
+     *
+     * @param newBitrate is a new bit rate to the audio.
      */
     public void setBitrate(final String newBitrate) {
         this.bitrate = newBitrate;
     }
+
     /**
-     * Obtains volume of audio converter
-     * @return a string of volume of audio converter
+     * Obtains volume of audio converter.
+     *
+     * @return a string of volume of audio converter.
      */
     public String getVolume() {
         return volume;
     }
 
     /**
-     * Sets new volume of the audio converter
-     * @param newVolume
+     * Sets new volume of the audio converter.
+     *
+     * @param newVolume is a new volumen to the audio.
      */
     public void setVolume(final String newVolume) {
         this.volume = newVolume;
     }
+
     /**
-     * Obtains the hz of audio converter
-     * @return a string of hz of audio converter
+     * Obtains the hz of audio converter.
+     *
+     * @return a string of hz of audio converter.
      */
     public String getHz() {
         return hz;
```

Comment:
If it possible, it would be good to change the name of hz, to a name more specific. Only a recommendation.

Reply:
ok

It was solved